File: buildSrc/src/test/resources/java/BrRegistrationApp.java
package com.mm.br.gateway.registration;

import com.mm.base.endpoint.MMApp;
import com.mm.br.gateway.registration.config.BrRegistrationConfig;

public class BrRegistrationApp {

  public static void main(String[] args) {
    MMApp.run("br", "br-registration", BrRegistrationConfig.class);
  }

}

File: buildSrc/src/test/resources/kt/BrRegistrationApp.kt
package com.mm.br.gateway.registration

import com.mm.base.endpoint.MMApp
import com.mm.br.gateway.registration.config.BrRegistrationConfig

object BrRegistrationApp {

  @JvmStatic
  fun main(args: Array<String>) {
    MMApp.run("br", "br-registration", BrRegistrationConfig::class.java)
  }
}